index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="keywords" content="Front-end web development">
+    <title>Arinze's Portfolio</title>
+    <link rel="shortcut icon" type="image/jpg" href="images/fav-icon.ico">
+</head>
+
+<body data-spy="scroll" data-target=".navbar" data-offset="50">
+    <div class="overall">
+        <header>
+            <nav id="nav-bar" class="navbar navbar-expand-lg navbar-dark navbar-inverse navbar-fixed-top">
+                <h4 class="navbar-brand" href="#">Arinze's <span>Design</span></h4>
+                <button class="navbar-toggler" type="button" data-toggle="collapse"
+                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav ml-auto">
+                        <li class="nav-item"><a class="nav-link active-link" href="#">Home</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#">Portfolio</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#about">About Me</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#social-media">Contact</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <main>
+            <section id="homepage">
+                <div class="home-text">
+                    <img src="images/picture.jpg" alt="personal picture">
+                    <p class="name">Hi! I'm <span>Arinze I.C</span></p>
+                    <p class="desc">Front-end web developer</p>
+                    <a href="#">My Work</a>
+                </div>
+            </section>
+    </div>
+    <section id="social-media">
+        <h3>Let's connect:</h3>
+        <div>
+            <a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i class="fab fa-instagram"></i></a>
```

Comment:
When you use target="_blank", always add rel="noopener" or rel="noreferrer". Visit https://developers.google.com/web/tools/lighthouse/audits/noopener for more information.